File: PDP/DrivingSchoolWebAPI3/DrivingSchoolAPIModels/ApiModels/LoginResponse.cs
namespace DrivingSchoolAPIModels
{
    /// <summary>
    /// Модель ответа при успешном входе в аккаунт
    /// </summary>
    public class LoginResponse
    {
        /// <summary>
        /// Токен для аутентификации
        /// </summary>
        public string Token { get; set; }
        /// <summary>
        /// Годность токена
        /// </summary>
        public DateTime Expiration { get; set; }
        public string Role { get; set; }
        public string Id { get; set; }
    }
}

File: PDP/DrivingSchoolAppAndroidClient/app/src/main/java/com/example/drivingschoolappandroidclient/api/models/LoginResponse.kt
package com.example.drivingschoolappandroidclient.api.models

import java.sql.Date

data class LoginResponse(
    val token: String,
    val expiration: Date,
    val role: String,
    val id: String,
) {
    val authHead get() = "Bearer $token"
}